libs/design/scss/theming/README.md

```diff
@@ -1,4 +1,4 @@
-# Theming
+# Daffodil Theming
 Daffodil's theming capabilities enables you to customize `@daffodil/design` components to reflect your brand. A theme consists of custom color configurations that will work in dark and light themes.
 
 ## Custom Colors
```

Comment:
There's a link here for `[validate-palette-function]` that goes nowhere, it should be removed.